CITATION.cff

```diff
@@ -0,0 +1,54 @@
+cff-version: 1.2.0
+title: scikit-hep-testdata
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+abstract: >-
+  Package providing example files for testing and developing packages against,
+  and simple helper methods to get larger files from common open-access data repositories.
+authors:
+  - family-names: "Das"
+    given-names: "Pratyush"
+    affiliation: "Purdue University"
+    orcid: "https://orcid.org/0000-0001-8140-0097"
+    email: "reikdas@gmail.com"
+  - family-names: "Eschle"
+    given-names: "Jonas"
+    affiliation: "University of Zurich"
+  - family-names: "Feickert"
+    given-names: "Matthew"
+    orcid: "https://orcid.org/0000-0003-4124-7862"
+    affiliation: "University of Wisconsin-Madison"
+  - family-names: Krikler
+    given-names: Benjamin
+    affiliation: RemotelyGreen
```

Comment:
Thanks for this! No need for an affiliation in my case (and it's not a required field in the CITATION.cff schema) but you could add my Orcid and / or github page:
```suggestion
    orcid: "https://orcid.org/0000-0001-9712-0030"
    website: "https://benkrikler.github.io"
```

Reply:
Perfect, was a best guess. Thanks @benkrikler.